daskhub/values.yaml

```diff
@@ -13,8 +13,13 @@ jupyterhub:
         description: "keras, flatbuffers, joblib, pillow, pytz, scikit-learn, scipy, uproot, root-numpy"
         kubespawner_override:
           image: eu.gcr.io/gke-dev-311213/jupyter-ml:20210518
+      - display_name: "PHYSLITE environment"
+        description: "environment for experiments with DAOD_PHYSLITE with uproot, awkward etc."
+        kubespawner_override:
+          image: <to-be-added>
     extraEnv:
       ATLAS_LOCAL_ROOT_BASE: /cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase
+      RUCIO_HOME: /srv/conda/envs/notebook
```

Comment:
not sure if i can also set this just for the PHYSLITE environment? It's needed to let rucio find the correct config file